supertokens_python/recipe/session/interfaces.py

```diff
@@ -78,16 +89,45 @@ async def create_new_session(
     ) -> SessionContainer:
         pass
 
+    @abstractmethod
+    def get_global_claim_validators(
+        self,
+        user_id: str,
+        claim_validators_added_by_other_recipes: List[SessionClaimValidator],
+        user_context: Dict[str, Any],
+    ) -> MaybeAwaitable[List[SessionClaimValidator]]:
+        pass
+
     @abstractmethod
     async def get_session(
         self,
         request: BaseRequest,
         anti_csrf_check: Union[bool, None],
         session_required: bool,
+        override_global_claim_validators: Optional[
+            Callable[
+                [SessionContainer, List[SessionClaimValidator], Dict[str, Any]],
+                MaybeAwaitable[List[SessionClaimValidator]],
+            ]
+        ],
         user_context: Dict[str, Any],
     ) -> Union[SessionContainer, None]:
         pass
 
+    @abstractmethod
+    async def assert_claims(
```

Comment:
The signature of this changed: now it only takes `claimvalidators: List[SessionClaimValidator]` instead of the callback and we build the list where we call `assertClaims` instead, to make overriding this function a bit easier.

Reply:
Just like the node PR, this has been done in a seperate PR 

https://github.com/supertokens/supertokens-python/pull/211